Ai Pin/Views/Notes/NotesView.swift

```diff
@@ -30,20 +33,23 @@ struct MemoryView: View {
 }
 
 struct NotesView: View {
+    @EnvironmentObject private var colorStore: ColorStore
     
     struct ViewState {
         var notes: [Memory] = []
         var isLoading = false
     }
     
-    @State
-    private var state = ViewState()
+    @State private var state = ViewState()
     
-    @Environment(NavigationStore.self)
-    private var navigationStore
+    @State private var selectedNoteId = ""
+    @State private var selectedNote = Note(text: "", title: "")
+    
+    @Environment(NavigationStore.self) private var navigationStore
```

Comment:
this will apply in a few places, but all property wrappers exist on their own line with what they are applied to below. This chunk of code should look like:

```swift
 @State 
 private var selectedNoteId = ""
 
 @State 
 private var selectedNote = Note(text: "", title: "")

 @Environment(NavigationStore.self) 
 private var navigationStore
    ```

Reply:
Noted!